conda_smithy/configure_feedstock.py

```diff
@@ -240,7 +240,40 @@ def _trim_unused_pin_run_as_build(all_used_vars):
         del all_used_vars["pin_run_as_build"]
 
 
-def _collapse_subpackage_variants(list_of_metas):
+def apply_migrations(list_of_metas, root_path):
+    """CFEP-9 variant migrations
+
+    Apply the list of migrations configurations to the build (in the correct sequence)
+    This will be used to change the variant within the list of MetaData instances,
+    and return the migrated variants.
+
+    This has to happend before the final variant files are computed.
+
+    The method for application is determined by the variant algebra as defined by CFEP-9
+
+    """
+    migrations_root = os.path.join(root_path, 'migrations', '*.yaml')
```

Comment:
Could we put this inside `.ci_support`?

Reply:
I believe we clear `.ci_support` at rerender time, so maybe not

Reply:
@scopatz, I don't think this was ever resolved. Please don't merge while there are pending conversations.

Reply:
@mariusvniekerk, I believe we can only clear top level .yaml files with a simple one line change